### DISCUSSION TITLE
Over 200 Styles - Preview

### DISCUSSION THREAD

**Discussion by 3Diva** [MoonRide303/Fooocus-MRE#165] [Show and tell]
I was inspired by @Elathi who did the "All 184 Styles" over on the Fooocus Vanilla discussion board: https://github.com/lllyasviel/Fooocus/discussions/143

And I wanted to do something similar for the styles available in Fooocus-MRE, since I know some of them are different. Here's over 200 styles. The base prompt is "a beautiful woman". I was unable to use the same seed as "Style Iterator" wouldn't seem to let me use the same seed. But these should at least be helpful as a quick reference for what each style generally does. If I missed any, please let me know!

![Fooocus-MRE - Styles 1-14](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/bb6f9fd2-ca9a-491c-970f-c695d897d20a)
![Fooocus-MRE - Styles 15-29](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/c6f759d6-03e9-40bc-9d47-21bfa5dcc9c0)
![Fooocus-MRE - Styles 30-44](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/8610f9bb-8a4b-462d-ac65-07eef5bbfb62)
![Fooocus-MRE - Styles 45-59](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/24827c71-c357-4ab1-b651-e0d5d8f24bb0)
![Fooocus-MRE - Styles 60-74](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/7d0ab62b-a65e-4bee-80a3-4587f27e1a57)
![Fooocus-MRE - Styles 75-84](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/d8c4eb8d-75c2-4632-9dcf-37e5452c4bf2)
![Fooocus-MRE - Styles 85-99](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/d403a004-0c58-4524-a6ea-b984e0101612)
![Fooocus-MRE - Styles 100-114](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/a6fa1ad1-c446-4ec6-a629-5d997519b4c0)
![Fooocus-MRE - Styles 115-129](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/6c8488e9-b4ed-476b-96d0-4ddcdac3409c)
![Fooocus-MRE - Styles 130-144](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/14e9387a-c85b-4950-9b70-cb661edd6d6c)
![Fooocus-MRE - Styles 145-159](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/4a47b8c8-cd7d-49eb-9a2e-88c7ba9fe4f0)
![Fooocus-MRE - Styles 160-174](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/d60482b1-0b7f-47d8-ba3d-a37d56aea190)
![Fooocus-MRE - Styles 175-182](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/d946630e-64b9-44df-b260-fdebae8c7d3b)

Three of this set got quite gruesome and very disturbing and I didn't know the policy of this site for images like that, so I thought it best to err on the side of caution and blur them out.
![Fooocus-MRE - Styles 183-197](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/c28e46dc-b3d9-4db4-9e52-0f2a950569aa)
![Fooocus-MRE - Styles 198-204](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/d04f812f-4706-45bf-801a-847bb70335dc)

I hope some of you find these reference images helpful. If I missed any, please let me know and I'll try and add any that I missed. I know there's at least one duplicate that slipped in as well. This was a project that took quite a few hours and it's far from perfect, but I thought it might be helpful for some to see a general idea of what the styles look like without having to generate over 200 images yourself to find out. 😄 

I hope you guys find some styles that inspire you!

**Comment by southbayjay** [comment]
Wow! Thank you! That's super helpful! Some really nice images there! I think I'll try this also with some other prompts this weekend.

**Reply by 3Diva** [reply]
That sounds like a fun weekend project! The "Style Iterator" makes it a LOT easier than manually loading each style. You can just click on Style Iterator and come back later to a TON of new images in your outputs folder! Be careful though, generating that many images in a row can be pretty hard on your system - particularly if you have a potato like mine! lol

**Reply by southbayjay** [reply]
Thanks for the info, I'll keep an eye on it. If I get a chance I'll share my results.

**Comment by 3Diva** [comment]
Have you ever wondered what an EXHAUSTED 1080 ti does? After generating hundreds of images in a row without a break I think my graphics card must have been really overheated or something. It did something quite strange. When I had it make one more image at the end of the day with the simple prompt of "a beautiful woman", it generates this:
![2023-10-10_19-38-04_7743](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/1fc4cdac-47e3-43f3-aab7-c56fe2706bb6)
...I think my system was trying to tell me it needed a break.

**Reply by southbayjay** [reply]
Oh my goodness, hopefully it's ok!

**Reply by 3Diva** [reply]
haha Thank you, I hope it's ok too! I need to be a bit more careful and maybe do projects like this more slowly and spread it out over a couple of days as to not tax my system too much. It was fun though, so as long as my computer is still ok I'm glad I did it! 😀

**Comment by MoonRide303** [comment]
@3Diva Nice job :). But I am not sure about this issue `I was unable to use the same seed as "Style Iterator" wouldn't seem to let me use the same seed.` - I just did quick test in MRE v2.0.78.5 on small set of styles with enabled same seed, and all the images were properly generated using same seed:
![image](https://github.com/MoonRide303/Fooocus-MRE/assets/130458190/369a4197-83a6-49da-8a5a-fef2a2a19289)
